packages/next/client/head-manager.ts

```diff
@@ -62,7 +62,7 @@ function updateElements(type: string, components: JSX.Element[]): void {
     i < headCount;
     i++, j = j!.previousElementSibling
   ) {
-    if (j!.tagName.toLowerCase() === type) {
+    if (j!.tagName?.toLowerCase() === type) {
```

Comment:
```suggestion
    if (j?.tagName.toLowerCase() === type) {
```
I think you have this in the wrong place since `previousElementSibling` can be null

Reply:
Thank you.